webui/src/views/OkrEditor.vue

```diff
@@ -3,7 +3,8 @@
         <header class="header">
             <h2 class="title">OKR Portal</h2>
 
-            <span class="user-name">{{user.name}}</span>
+            <span class="user-name" @click="logOut">Sign Out</span>
+            <span class="user-name" >{{user.name}}</span>
```

Comment:
Here also add `header-nav-item` class and remove extra space. So, in `main.less` file also change `user-name` to `header-nav-item`